pipeline-adapters/mlrun-pipelines-kfp-common/pyproject.toml

```
@@ -0,0 +1,39 @@
+[project]
+name = "mlrun-pipelines-kfp-common"
+version = "0.3.13"
+description = "MLRun Pipelines package for providing KFP common functionality"
```

Unrelated to the change but I think adding the "adapter" adjective makes sense to all packages
```suggestion
description = "MLRun Pipelines adapter package for providing KFP common functionality"
```